src/axom/primal/examples/primal_introduction.cpp

```diff
@@ -71,21 +71,20 @@
 
 // _using_start
 // "using" directives to simplify code
-using namespace axom;
-using namespace primal;
+namespace primal = axom::primal;
 
 // almost all our examples are in 3D
 constexpr int in3D = 3;
 
 // primitives represented by doubles in 3D
```

Comment:
While I was editing the primal tests, I modernized them a bit, e.g. type aliases via `using` instead of `typedef` and using the initializer-list constructors for primal's primitives instead of `make_point`, `make_vector`, ....